Lib/test/test_site.py

```diff
@@ -277,6 +284,11 @@ def test_getusersitepackages(self):
         self.assertEqual(site.USER_BASE, site.getuserbase())
 
     def test_getsitepackages(self):
+        sys.path.append(os.path.abspath(os.path.join(__file__, '..', 'vendor_config')))
+        # force re-load of vendor schemes with the patched sys.path
+        site._VENDOR_SCHEMES = None
+        sysconfig._load_vendor_schemes()
+
```

Comment:
Would it make sense to have a separate test rather than mutate an existing test?

Reply:
Sure, we still need the test as-is, but we can also keep the original one.